### DISCUSSION TITLE
Deploy Help

### DISCUSSION THREAD

**Discussion by M23jJohn** [pocketbase/pocketbase#5802] [Q&A]
Can I ask for guides in deploying my pocketbase folder (with pb_data) on **render** using docker especially with _Data persisting_ part

**Comment by M23jJohn** [comment]
can anyone help me

**Comment by jonathan-martz** [comment]
https://dev.to/_russell/deploying-pocketbase-with-docker-nginx-and-ssl-323l

You could check out this one.

Pocketbase, Docker,SSL Cert everything what you need for an instance.
